scripts/pdf/render.py

```diff
@@ -17,7 +17,7 @@
 from weasyprint import HTML
 
 
-def main(loc, colorscheme):
+def main(loc, colorscheme, output_filename=None):
```

Comment:
This is an internal script, I think it's fine to make breaking changes to function signatures and require people use it with three arguments, given it's how we always use it here.

```suggestion
def main(loc, colorscheme, output_filename):
```